consoles/VNC.pm

```diff
@@ -225,7 +225,11 @@ sub _handshake_security {
             $number_of_security_types = unpack('C', $number_of_security_types);
         }
         if ($number_of_security_types == 0) {
-            die 'Error authenticating';
+            my $generic_error_message = 'Error authenticating: server returned not "security types"';
```

Comment:
What does that mean? Maybe it's a typo but my head can't really parse the message

Reply:
It should be no instead of not. I actually already fixed that but forgot to push the amended commit.